engines/director/detection_tables.h

```diff
@@ -327,6 +340,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "searchlearn",     	"Search & Learn Adventures" },
 	{ "secretkeys",     	"Search for the Secret Keys with Mickey" },
 	{ "secretpaths1",		"Secret Paths in the Forest" },
+	{ "secretsluxor",		"Secrets of the Luxor" },
```

Comment:
Can you make this just `luxor`? (sorry, just a preference)

Reply:
Done